perception/multi_object_tracker/src/multi_object_tracker_core.cpp

```diff
@@ -386,6 +386,15 @@ void MultiObjectTracker::sanitizeTracker(
         continue;
       }
 
+      if (
+        object1.shape.type == autoware_auto_perception_msgs::msg::Shape::POLYGON &&
+        tier4_autoware_utils::getArea(object1.shape) == 0 &&
```

Comment:
It is not very desirable to make equal comparisons between floating-point types and direct numbers. 🙏 
In this case, it is 0, so it is OK, but it may cause unintended bugs.

Reply:
reference : https://www.geeksforgeeks.org/problem-in-comparing-floating-point-numbers-and-how-to-compare-them-correctly/

Reply:
@yukkysaito can we use the following directly directly:

https://github.com/autowarefoundation/autoware.universe/blob/52b2fd6a2099f0ff2fc0e9b9c22a9a14163ad849/common/autoware_auto_common/include/autoware_auto_common/helper_functions/float_comparisons.hpp#L106-L122

Or do we have these float comparisons somewhere else?